docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -39,12 +39,27 @@ These permissions are a baseline access for the ACS:CS team across OSD/ROSA infr
 ## backplane-acs-admins-cluster: `cluster`
 Permissions at cluster scope.
 
+* view events
 * view projects
 * view nodes
 * view pods
+* view logs
```

Comment:
```suggestion
* view pod logs
```

There is some risk of secret data making it into a log, though probably very low risk.  Is there some set of namespaces that this permission could be bound to instead of cluster scope?

Reply:
It's hard to know honestly, because we're often looking at a wide range of openshift/rhacs components logs, depending on what feature needs to be implemented, or what problem might occur.